packages/interactor/src/resolve.ts

```diff
@@ -7,7 +7,13 @@ const defaultSelector = 'div';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function findMatches(parentElement: Element, interactor: Interactor<any, any, any>): Match<Element, any>[] {
-  let elements = Array.from(parentElement.querySelectorAll(interactor.specification.selector || defaultSelector));
+  let elements;
+  if(interactor.specification.selector === ':root') {
+    // this is a bit of a hack, because otherwise there isn't a good way of selecting the root element
+    elements = [parentElement.ownerDocument.querySelector(':root')];
+  } else {
+    elements = Array.from(parentElement.querySelectorAll(interactor.specification.selector || defaultSelector));
+  }
```

Comment:
To clarify *why* I had to do this: for an interactor without a parent, the `parentElement` will be the HTML element, and there is no good way in CSS to select the element *itself* via a selector. We'd essentially want to do something like `querySelector('self')`. The closest is the selector `:scope`, which is roughly meant to do this, but support for it is spotty. JSDOM doesn't support it and neither does IE.

The alternative is to change the type of `Element` to be `Element | Document` instead, and use the document as the root instead. This would be a more elegant solution, but it would make the already complicated types even more complicated and probably add a bunch of useless typing stuff and if statements, just for this one exception.